client/signup/index.web.js

```diff
@@ -9,13 +9,10 @@ import page from 'page';
 import controller from './controller';
 import { makeLayout, render as clientRender } from 'calypso/controller';
 import { getLanguageRouteParam } from 'calypso/lib/i18n-utils';
-import { loadExperimentAssignment } from 'calypso/lib/explat';
 
 export default async function () {
 	const lang = getLanguageRouteParam();
 
```

Comment:
I think it is a good idea to add a non-awaited `loadExperimentAssignment('refined_skin_v1')` in here as well - this will prefetch the experiment while everything else is loading so there will be less chance of a loading state occuring.

Reply:
Ah good point, I'll do that.

Reply:
I've implemented this change. The PR is ready for review once again :)